amazonfreertossdk/src/main/java/software/amazon/freertos/amazonfreertossdk/AmazonFreeRTOSConstants.java

```diff
@@ -78,7 +78,10 @@
     }
 
     public enum AmazonFreeRTOSError {
-        BLE_DISCONNECTED_ERROR
+        BLE_NO_ERROR,
```

Comment:
initialize to zero to ensure BLE_NO_ERROR is set to 0, when adding more error codes ?

Reply:
The error value is only used in transport. The phone and MCU don't have hard-coded integers that map to a certain enum. Currently `Enum.ordinal()` is being used so the `ith` declared enum member has value `i` (0-indexed) -- similar to default enum evalutaion in C